test/test_torchinductor.py

```diff
@@ -1255,6 +1256,50 @@ def fn(x, w, b):
             check_lowp=False,
         )
 
+    @unittest.skipIf(HAS_CUDA, "only support cpu channels_last")
```

Comment:
Is there a reason this needs to be CPU only?  Does it error if you enable it for GPU?

Reply:
This PR only enables CPU channels last, for GPU, it will be failed, for GPU path, I find that the channels last depend on cudnn version, see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ConvUtils.h, I need to find a method to check the version at python side, and then create another PR to fix GPU path.